mlflow/requirements.txt

```diff
@@ -0,0 +1,11 @@
+mlflow
+pydantic
+starlette
+limits
+watchfiles
+fastapi
+slowapi
+uvicorn
+aiohttp
+gunicorn
+boto3
```

Comment:
Can we pin versions to each package? Particularly important with this being the first non-poetry managed service.

Reply:
Pinned versions to each package